spec/system/user_show_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'User Show Page', type: :system do
+  before do
```

Comment:
- [ ] Please you a missing a test that tests that a user redirects to the post's show page when you click on a user's post. :blush: